pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BrokersImpl.java

```diff
@@ -42,14 +42,24 @@ public BrokersImpl(WebTarget web, Authentication auth, long readTimeoutMs) {
         adminBrokers = web.path("admin/v2/brokers");
     }
 
+    @Override
+    public List<String> getActiveBrokers() throws PulsarAdminException {
+        return sync(() -> getActiveBrokersAsync(null));
+    }
+
+    @Override
+    public CompletableFuture<List<String>> getActiveBrokersAsync() {
+        return getActiveBrokersAsync(null);
+    }
+
     @Override
     public List<String> getActiveBrokers(String cluster) throws PulsarAdminException {
         return sync(() -> getActiveBrokersAsync(cluster));
     }
 
     @Override
     public CompletableFuture<List<String>> getActiveBrokersAsync(String cluster) {
-        WebTarget path = adminBrokers.path(cluster);
+        WebTarget path = cluster == null ? adminBrokers : adminBrokers.path(cluster);
```

Comment:
Seems no need to modify

Reply:
`WebTarget.path` does not support null param, othewise it`ll get "path is 'null'."